src/test/scala/io/iohk/ethereum/blockchain/sync/SyncControllerSpec.scala

```diff
@@ -993,13 +1010,10 @@ class SyncControllerSpec extends FlatSpec with Matchers {
 
     val peer = Peer(new InetSocketAddress("127.0.0.1", 0), peerTestProbe.ref, false)
 
-    time.advance(1.seconds)
+    Thread.sleep(1.seconds.toMillis)
```

Comment:
maybe we can pick one version 
`Thread.sleep(2000)` / `Thread.sleep(1000)`
or
`Thread.sleep(2.seconds.toMillis)` / `Thread.sleep(1.seconds.toMillis)`
I am for second version